hasher-matcher-actioner/webapp/src/components/settings/ActionPerformer/WebhookActioner.tsx

```diff
@@ -5,15 +5,27 @@
 import React from 'react';
 import Card from 'react-bootstrap/Card';
 import Form from 'react-bootstrap/Form';
-import {PropTypes} from 'prop-types';
+import PropTypes from 'prop-types';
+
+export interface Map {
+  [key: string]: string;
+}
```

Comment:
Dont think we need this interface if the two uses are unrelated. Can we change to two `type`s instead?